docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -151,6 +151,15 @@ POST /druid/indexer/v1/supervisor
 ```
 Use `Content-Type: application/json` and provide a supervisor spec in the request body.
 
+Calling this endpoint when there is already an existing supervisor for the same dataSource will cause:
+
+- The running supervisor to signal its managed tasks to stop reading and begin publishing.
+- The running supervisor to exit.
+- A new supervisor to be created using the configuration provided in the request body. This supervisor will retain the
+existing publishing tasks and will create new tasks starting at the offsets the publishing tasks ended on.
```

Comment:
@nishantmonu51 no it won't wait for the tasks to finish publishing, it will wait until it has successfully signaled the tasks to begin publishing and then will exit